c7n/resources/connect.py

```diff
@@ -0,0 +1,58 @@
+from c7n.manager import resources
+from c7n.query import QueryResourceManager, TypeInfo
+from c7n.filters import ValueFilter
+from c7n.utils import local_session, type_schema
+
+
+@resources.register('connect-instance')
+class Connect(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        service = 'connect'
+        enum_spec = ('list_instances', 'InstanceSummaryList', None)
+        arn_type = 'connect'
+        name = id = 'InstanceId'
+        universal_taggable = object()
```

Comment:
afaics this resource/service isn't supported by universal tagging, so this field should be omitted.